### DISCUSSION TITLE
Sending message from studio to websocket service.

### DISCUSSION THREAD

**Discussion by tsuang** [foxglove/community#30] [Q&A]
Is it possible to send message from foxglove studio to websocket service? I would like to build a custom panel that allow operator to manually move the robot. 
![image](https://user-images.githubusercontent.com/69434036/184005157-c0045149-730d-4488-9be2-82ed3b86a2ef.png)

**Comment by jtbandes** [comment]
See this issue: https://github.com/foxglove/ws-protocol/issues/38

**Reply by tsuang** [reply]
Thanks, this one seem to solve the problem: https://github.com/foxglove/ws-protocol/pull/158#issue-1245740836
